src/ppx/application.cpp

```diff
@@ -652,31 +652,6 @@ void Application::DispatchInitKnobs()
 void Application::DispatchConfig()
 {
     Config(mSettings);
-
-    if (mSettings.appName.empty()) {
```

Comment:
I guess taking this out is a slight change from the original behaviour, since DispatchConfig can be overriden by applications, and now it can't modify this. But it might be for the best